src/Silverback.Integration/Messaging/Diagnostics/ActivityTagNames.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+namespace Silverback.Messaging.Diagnostics
+{
+    /// <summary>
+    ///     Contains the Tag Names of Tags added to the Activities.
+    /// </summary>
+    public static class ActivityTagNames
+    {
+        /// <summary>
+        /// Tag Name for the Message Identification. The Tag is written in the form [key]@[Value].
```

Comment:
As mentioned already, this is true for Kafka only. MQTT will be different and the identifier is just made-up in the consumer, having no real meaning for the distributed trace.